### PR TITLE
suppress version date in sqlite3

### PR DESCRIPTION
**Pull request by rongxin-liu**

### CODE REVIEW

**Comment by dmalan** [comment]
So the version will also get out of date eventually. Should we just remove `SQLite version.*`?

**Comment by rongxin-liu** [comment]
I'm neutral about this. Since SQL commands are unlikely to change frequently, not displaying the version number seems fine.

**Comment by dmalan** [comment]
Per https://github.com/cs50/cli/issues/157, perhaps cleanest to suppress "usage hints" too, a la `python -3`?

**Comment by rongxin-liu** [comment]
In that case, best to just comment out that particular print statement in source code entirely.

```
      printf(
        "SQLite version %s %.19s\n" /*extra-version-info*/
        "Enter \".help\" for usage hints.\n",
        sqlite3_libversion(), sqlite3_sourceid()
      );
```

**Comment by dmalan** [comment]
Want to create a `.patch` file that we can include in the repo, which the Dockerfile runs on `shell.c`? And when we update `sqlite3` next, we can ensure the patch still works?

**Comment by rongxin-liu** [comment]
> Want to create a `.patch` file that we can include in the repo, which the Dockerfile runs on `shell.c`? And when we update `sqlite3` next, we can ensure the patch still works?

How does this work? Could we update the `sed` command in Dockerfile to comment out those lines?

**Comment by rongxin-liu** [comment]
You mean place the `sed` command inside `.patch` file, `ADD` that script to Dockerfile and then run it during the build process?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .*
 !/.github/
 !.gitignore
+!.patch
```
Comment (dmalan) [inline .gitignore:4]: It _might_ be a convention to call this `shell.c.patch` perhaps?

Comment (dmalan) [reply .gitignore:4]: (In which case no need for `.gitignore`.)

File: Dockerfile [hunk 2]
```diff
@@ -89,14 +89,17 @@ RUN apt update && \
 # Install SQLite 3.x
 # https://www.sqlite.org/download.html
 # https://www.sqlite.org/howtocompile.html#compiling_the_command_line_interface
+ADD .patch /tmp
```
Comment (dmalan) [inline Dockerfile:92]: I think `COPY` tends to be best, per https://stackoverflow.com/a/24958548. (And what we use elsewhere.)

**Review by dmalan — CHANGES_REQUESTED** [review]

**Review by rongxin-liu — COMMENTED** [review]
To summarize, we will now be commenting out the print statement in source code `shell.c` to suppress version number and usage hint. We create a patch and apply it to the newly downloaded shell.c when building the cli image.

**Pull request merged**